utilities/storage.py

```diff
@@ -634,7 +634,7 @@ def write_file(vm, filename, content, stop_vm=True):
         vm.stop(wait=True)
 
 
-def run_command_on_cirros_vm_and_check_output(vm, command, expected_result):
+def run_command_on_vm_and_check_output(vm, command, expected_result):
```

Comment:
Please create a new function under `tests/storage/snapshots/utils.py`
Upgrade tests will be changed separately.

Please try using `run_ssh_commands` instead of console. You can see an example in `test_vmexport_snapshot_manifests` (in vmexport test it only validates the content of one file, but for the snapshot tests we want to keep the validation of the `ls` command output).